CommonApi/src/main/java/mezz/jei/api/recipe/IRecipeManager.java

```diff
@@ -111,13 +113,15 @@ public interface IRecipeManager {
 	 * Returns a drawable recipe layout, for addons that want to draw the layouts somewhere.
 	 *
 	 * @param recipeCategory the recipe category that the recipe belongs to
+	 * @param decorators	 the decorators for the recipe category.
 	 * @param recipe         the specific recipe to draw.
 	 * @param focusGroup     the focuses of the recipe layout.
 	 *
-	 * @since 11.5.0
+	 * @since 15.1.0
 	 */
 	<T> Optional<IRecipeLayoutDrawable<T>> createRecipeLayoutDrawable(
 		IRecipeCategory<T> recipeCategory,
+		Collection<IRecipeCategoryDecorator<T>> decorators,
```

Comment:
I think we can avoid changing this and avoid exposing `getRecipeCategoryDecorators`, they can be applied internally all the time.

Reply:
Not exactly sure what you mean by that.

Reply:
I think we can implement this logic internally without making any changes to `IRecipeManager` in the api

Reply:
Since I am not sure what you have in mind for that, feel free to add it to the branch yourself. Maybe together with the `Map<RecipeType, RecipeCategory>` idea from the other review. :)

Reply:
Sounds good. Can you give me push access to this branch?
I think you have to select "Allow edits from maintainers" on the PR

Reply:
That usually is enabled by default. I can't seem to find a setting where you can set this after the PR is open already. This may be due to the fact that we forked it into our organization.
I'll just add it as an outside collaborator for the forked repo as an alternative solution.

Reply:
Thanks, that worked. I have pushed my changes, please let me know if this still looks good to you.